Fix/content type gen windows (#9877)
withastro/astro: .changeset/funny-mice-help.md

---
"astro": patch
---

> Fix content type reference to content config on windows.

Reviewer: ```suggestion
Fixes the content config type path on windows
```
